roles/metal3/defaults/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 metal3_repo_url: https://github.com/suse-edge/charts.git
 metal3_branch: main
-metal3_chart_version: 0.6.5
+
+metal3_chart_version: 0.7.2
```

Comment:
I guess this is include also the reprovisioning patch at the same time, right? But I'm wondering if we can make it already available.

Reply:
Yes I aligned with the latest upstream version ref https://github.com/suse-edge/charts/pull/133 even though the downstream release of this is still pending